packages/api-aco/src/folder/folder.crud.ts

```diff
@@ -66,6 +66,9 @@ export const createFolderCrudMethods = ({ storageOperations }: CreateAcoParams):
             await storageOperations.deleteFolder({ id });
             await onFolderAfterDelete.publish({ folder });
             return true;
+        },
+        async getHierarchyById(id: string) {
+            return storageOperations.getFolderHierarchyById({ id });
         }
```

Comment:
This can be replaced by the logic described in the following comments. Basically you don't need a new storage operations method, because you already have all the data you need.